.stylelintrc

```
@@ -0,0 +1,70 @@
+{
+  "extends": [
+    "stylelint-config-standard-scss",
+    "stylelint-config-idiomatic-order"
+  ],
+  "plugins": [
+    "stylelint-declaration-strict-value",
+    "@stylistic/stylelint-plugin"
+  ],
+  "overrides": [
+    {
+      "files": ["src/assets/scss/*.{css,scss}"],
+      "customSyntax": "postcss-scss",
+      "rules": {
+        "media-feature-range-notation": "prefix",
+        "color-function-notation": "legacy",
+        "shorthand-property-no-redundant-values": null,
+        "declaration-block-no-redundant-longhand-properties": [true, {
+          "ignoreShorthands": [
+            "inset",
+            "inset-block",
+            "inset-inline",
+            "mask",
+            "overscroll-behavior",
+            "padding-block",
+            "padding-inline",
+            "place-content",
+            "place-items",
+            "place-self",
+            "scroll-margin",
+            "border-block",
+            "border-block-end",
+            "border-block-start",
+            "border-inline",
+            "border-inline-end",
+            "border-inline-start"
+          ]
+        }],
+        "no-invalid-position-at-import-rule": null,
+        "no-descending-specificity": null,
+        "selector-class-pattern": null,
+        "scss/dollar-variable-pattern": null,
+        "selector-id-pattern": null,
+        "scss/no-global-function-names": null,
+        "selector-type-no-unknown": null,
+        "property-no-vendor-prefix": [true,
+          {
+            "ignoreProperties": ["backdrop-filter", "user-select", "initial-letter", "text-decoration", "text-stroke", "text-fill-color", "line-clamp", "box-decoration-break", "stretch"]
```

Only `user-select` is necessary now

```suggestion
            "ignoreProperties": ["user-select"]
```

---

I would like to keep them in. Even if they are not necessary now, I want to set up the style lint so we don't have to remember to add these back if we use one of these properties.

---

Gotcha. Were did this list come from?

---

I've got the list from Fully Stacked at https://fullystacked.net/prefix/

---

That's a great reference, thanks! I know I suggested making this a JSON file, but it could be even better to make it `stylelint.config.js` so that we can add comments and mention where this list came from. Or at least note that those are the vendor prefixes that are still needed as of November 2024 when that article was last updated. I think that could be helpful as this list will presumably shrink over time.